### ISSUE TITLE
Underthesea 6.8.3 requires fasttext on import but the dependency is missing

### ISSUE THREAD

**Issue by BLKSerene** [undertheseanlp/underthesea#740]
When importing `Underthesea` 6.8.3:

```
Traceback (most recent call last):
  File "<pyshell#0>", line 1, in <module>
    import fasttext
ModuleNotFoundError: No module named 'fasttext'
```

There is no issue with `Underthesea` 6.8.0

**Comment by rain1024** [comment]
@BLKSerene, which module did you use, and can you share your code?

**Comment by BLKSerene** [comment]
A simple import:

```
import underthesea
```

**Comment by BLKSerene** [comment]
Seems that this has been fixed by the re-released 6.8.3.

**Comment by rain1024** [comment]
Actually, you're correct. 

Without fasttext, when import underthesea, it show unwanted error log

```
>>> import underthesea
No module named 'fasttext'
```
